### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonaslagoni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [COMMAND HERE]
   Options:
 
     -V, --version                  output the version number
-    -w, --watch                    watches the the template directory and the async api document for change, and re-generate the files when they occour.
+    -w, --watch                    watches the template directory and the async api document for change, and re-generate the files when they occour.
```
Comment (fmvilas) [inline README.md]: Now that you're here, can you please fix "async api"? It should be "AsyncAPI" because it's a brand name.

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [COMMAND HERE]
   Options:
 
     -V, --version                  output the version number
-    -w, --watch                    watches the the template directory and the async api document for change, and re-generate the files when they occour.
+    -w, --watch                    watches the template directory and the AsyncAPI document for change, and re-generate the files when they occour.
```
Comment (fmvilas) [inline README.md]:
```suggestion
    -w, --watch                    watches the template directory and the AsyncAPI document for change, and re-generate the files when they occur.
```

File: README.md [hunk 3]
```diff
@@ -57,9 +57,7 @@ ag -o ./docs --param title='Hello from param' asyncapi.yaml markdown
 In the template you can use it like this: ` {{ params.title }}`
 
 ### File templating
-To separate functionality into files, one can specify a filename like `$$channel$$.js` to generate a file for each channel defined in your specification. The following separations are supported: `$$channel$$`, `$$message$$`, `$$schema$$`, `$$parameter$$` and `$$securityScheme$$`.
-
-> As of right now `$$message$$`, `$$schema$$`, `$$parameter$$` and `$$securityScheme$$` are only supported if it is defined in the component section ([#170](https://github.com/asyncapi/generator/issues/170)).
+To separate functionality into files, one can specify a filename like `$$channel$$.js` to generate a file for each channel defined in your async api document. The following separations are supported: `$$channel$$`, `$$message$$`, `$$schema$$`, `$$parameter$$` and `$$securityScheme$$`.
```
Comment (fmvilas) [inline README.md]:
```suggestion
To separate functionality into files, one can specify a filename like `$$channel$$.js` to generate a file for each channel defined in your AsyncAPI document. The following separations are supported: `$$channel$$`, `$$message$$`, `$$schema$$`, `$$parameter$$` and `$$securityScheme$$`.
```

**Comment by jonaslagoni** [comment]
`-o, --output <outputDir>       directory where to put the generated files (defaults to current directory) (default: /Users/fmvilas/www/asyncapi-generator)`

Why do we have two defaults 🤔

**Comment by fmvilas** [comment]
LOL, that's true. And it has my system path. Would you mind deleting it?

**Comment by jonaslagoni** [comment]
Done 👍

**Pull request merged**